docs/supplement.rst

```diff
@@ -1,32 +1,105 @@
+
+Report review
+^^^^^^^^^^^^^
+
+The on-call ACA reviewer inspects the reports during the normal work week,
+checking that the processing and indicated magnitude updates appear reasonable.
+Most of the time this is the case and no further review is required.
+
+For star observations that are suspect or are otherwise unusual, a disposition
+will be made based on the report and by examining star image telemetry or other
+available data as needed. Dispositions may include:
+
+- Exclude or include specific star observations from the magnitude estimation.
+  This informs the processing code about whether the data from a star
+  observation can be used to compute the estimated star magnitude.
+- Add the star to the AGASC supplement bad star table. This prevents it from
+  being used in star selection and is independent of the magnitude estimate.
+- Manually set a star magnitude to a given value in the case that the
+  automated code is unable to determine a value. The most common case is where a
+  guide star is never acquired despite repeated search attempts by OBC
+  commanding. In this case the magnitude would typically be be set to the
+  catalog ``MAXMAG`` to indicate a lower limit to the magnitude, and the star
+  would normally be added to the bad stars table.
+
+Other team members may be consulted at the discretion of the ACA reviewer.
+
+Applying review
+^^^^^^^^^^^^^^^
+
+If no star dispositions were required then this step does not apply.
+
+After deciding on the status of suspect star observations, the ACA reviewer
+edits ``$SKA/data/agasc/obs-status.yml`` accordingly. In order to apply the
+changes the reviewer runs::
+
+  agasc-supplement-tasks disposition
 
-Additionally, the table of magnitudes in the supplement is updated regularly. Whenever the magnitudes are updated, a
-report is placed in `agasc/supplement_reports <https://cxc.cfa.harvard.edu/mta/ASPECT/agasc/supplement_reports>`_.
-Upon reviewing these reports we can decide to exclude/include specific star observations from the magnitude estimation.
-To accomplish this, we need to add the corresponding observations to the "obs" table in the supplement.
-The approval procedure for this update is not yet specified, but after approval the supplement will be updated as
-detailed below.
+This command updates the release candidate version of the supplement in
+``$SKA/data/agasc/rc`` as well as the weekly reports. The ACA reviewer then
+checks the updated reports to validate that any dispositions have been applied
+correctly.
+
+Promotion
+^^^^^^^^^
+
+Once satisfied, the ACA reviewer runs the following to *schedule* the update
+for promotion on the following Sunday::
+
+  agasc-supplement-tasks promote
+
+The scheduling is done in that tool using a linux ``at`` command on the HEAD
+``kady`` machine as the ``aca`` user. In the unlikely event needing to cancel
+the promotion, inspect the ``at`` queue and remove the job.
```

Comment:
the current implementation does not use `at` but copies the files into `$SKA/data/agasc/rc/promote/`. The idea is that there is a regular cron job that copies from this directory. Promotion is therefore two steps: stage + promote. What the reviewer does is to "stage".

I did it this way because both you and Jean prefer to have this configured with the crontabs, so it can be changed without changing the code and cutting a release. Using `at` in the script would mean we have to cut a release if we want to change when we do the promotion.

Also... if I were to do it in this way, I would not use `at` but would set the time in the promotion task schedule, and there would be only one step. No need for staging.

Reply:
another option is to use `at` and let the reviewer choose the time.

Reply:
Perfect, I actually preferred the `promote` area idea but thought that I got overruled at one of our meetings. :smile:  You can see I didn't look at the code...  I like the `promote` area especially because it is easy to see the status and trivial to cancel a pending promotion.